core/vm/message/message_signer.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_MESSAGE_SIGNER_HPP
+#define CPP_FILECOIN_CORE_VM_MESSAGE_SIGNER_HPP
+
+#include <gsl/span>
+
+#include "vm/message/message_codec.hpp"
+
+namespace fc::vm::message {
+
+  using PrivateKey = gsl::span<uint8_t>;
+  using PublicKey = gsl::span<uint8_t>;
+
+  /**
+   * Interface of an UnisgnedMessage Signer.
+   */
+  class MessageSigner {
```

Comment:
According https://filecoin-project.github.io/specs/#system-requirements private keys must be secret, that's why we have KeyStore with `sign()` and `verify()` methods. I cannot find usage of MessageSigner, imho we should use KeyStore for signing.